addons/payment/data/payment_method_data.xml

```diff
@@ -877,44 +946,44 @@
         />
     </record>
 
-    <record id="payment_method_clearpay" model="payment.method">
-        <field name="name">Clearpay</field>
-        <field name="code">clearpay</field>
+    <record id="payment_method_cimb_niaga" model="payment.method">
+        <field name="name">CIMB Niaga</field>
+        <field name="code">cimb_niaga</field>
         <field name="sequence">1000</field>
         <field name="active">False</field>
-        <field name="image" type="base64" file="payment/static/img/clearpay.png"/>
+        <field name="image" type="base64" file="payment/static/img/cimb_niaga.png"/>
         <field name="support_tokenization">False</field>
         <field name="support_express_checkout">False</field>
-        <field name="support_refund">partial</field>
+        <field name="support_refund">none</field>
         <field name="supported_country_ids"
                eval="[Command.set([
-                         ref('base.uk'),
+                         ref('base.id'),
                      ])]"
         />
         <field name="supported_currency_ids"
                eval="[Command.set([
-                         ref('base.GBP'),
+                         ref('base.IDR'),
                      ])]"
         />
     </record>
 
-     <record id="payment_method_cimb_niaga" model="payment.method">
-        <field name="name">CIMB Niaga</field>
-        <field name="code">cimb_niaga</field>
+    <record id="payment_method_clearpay" model="payment.method">
```

Comment:
From the upgrade PR. Are you just swapping the order of clearpay and cimb niaga or there is an actual change happening here?

Reply:
Just swapping, they were all in alphabetical order except that one, it bothered me... a lot. (I'm weak xD)